Dockerfile.utils

```diff
@@ -21,7 +21,8 @@ RUN apk add --no-cache socat tar
 
 COPY --from=builder /workspace/func-util /usr/local/bin/
 RUN ln -s /usr/local/bin/func-util /usr/local/bin/deploy && \
-    ln -s /usr/local/bin/func-util /usr/local/bin/scaffold
+    ln -s /usr/local/bin/func-util /usr/local/bin/scaffold && \
+    ln -s /usr/local/bin/func-util /usr/local/bin/s2i
```

Comment:
how to we ensure this is up-to-date?

Is there any "harm" vendoring this extra binary?

Reply:
Actually this ensures the s2i in the binary and the util image is the same which is nice. If there is any updated required (e.g. ther is CVE) we do it just in go.mod and it updates it both. What are disadvantages?

Reply:
I was worried for carrying on CVEs, but looks like it is addressed

Reply:
Well we basically already ship s2i binary. We use it as an library in `func` CLI.

Reply:
>Well we basically already ship s2i binary. We use it as an library in func CLI.

So we are already on hook for keeping it up to date.